warehouse/packaging/models.py

```diff
@@ -107,6 +112,13 @@ def __getitem__(self, version):
             raise KeyError from None
 
     @property
+    def latest_release(self):
+        try:
+            return self.releases.order_by('created')[0]
```

Comment:
This should have a ``.limit(1)`` added to it and should use ``.first()`` instead of trying to index the returned list. There is likely a better name for this too I think, "latest" is overloaded, is it the most recently released version or the highest version number available?